app/controllers/my-services/get-index-controller.js

```diff
@@ -8,53 +8,61 @@ const { response } = require('../../utils/response')
 const serviceService = require('../../services/service_service')
 const getHeldPermissions = require('../../utils/get_held_permissions')
 
-module.exports = (req, res) => {
-  const servicesRoles = _.get(req, 'user.serviceRoles', [])
-  const newServiceId = _.get(req, 'query.s')
+const hasAccountWithPayouts = function hasLiveStripeAccount (gatewayAccounts) {
+  return gatewayAccounts.some(gatewayAccount =>
+    !isDirectDebitAccount(gatewayAccount) &&
+    gatewayAccount.payment_provider === 'stripe' &&
+    gatewayAccount.type === 'live')
+}
 
-  const displayMyServices = servicesData => {
-    const data = {
-      services: servicesData,
-      services_singular: servicesData.length === 1,
-      env: process.env
-    }
-    if (newServiceId) {
-      servicesData.filter(serviceData => {
-        return serviceData.external_id === newServiceId
-      }).forEach(service => {
-        data.new_service_name = service.name
-      })
-    }
+const isDirectDebitAccount = function isDirectDebitAccount (gatewayAccount) {
```

Comment:
I believe this helper already exists (for some reason) in the direct debit connector client, there are other places in the code that we use that so I think it would be good to use it here. (Understood that this is just a good moving around of the legacy code though!)